cloudshell/networking/cisco/command_actions/add_remove_vlan_actions.py

```diff
@@ -29,7 +36,16 @@ def __init__(self, cli_service, logger):
         self._logger = logger
 
     @staticmethod
-    def verify_interface_configured(vlan_range, current_config):
+    def verify_interface_configured(current_config):
+        """Verify interface configuration.
+
+        :param current_config:
+        :return: True or False
+        """
+        return AddRemoveVlanActions.CHECK_VLAN_ASSIGNED.search(current_config)
```

Comment:
it'll return match or None not True or False
you can write it like this:
```suggestion
        return bool(AddRemoveVlanActions.CHECK_VLAN_ASSIGNED.search(current_config))
```